src/actor/Verifier.ts

```diff
@@ -182,13 +185,34 @@ async function verifyPublicPresentation(
   return { verified, claims: verified ? attestedClaims : [] }
 }
 
+/**
+ * Checks whether each value is either an instance of an Accumulator or a [[PublicAttesterIdentity]].
+ *
+ * @param values Either an array or undefined which should be checked.
+ * @param isAcc A boolean to display whether we check for Accumulators.
+ * @returns True if the supplied values are correct.
+ */
+function verificationCheck(
+  values:
+    | Array<gabi.Accumulator | PublicAttesterIdentity | undefined>
+    | undefined,
```

Comment:
what is the reason this is undefined and not `unknown` ? That's usually what you would use on a type guard, isn't it.

Reply:
Correct, fixed in 5db18c9